pkg/vm/vm_test.go

```diff
@@ -729,6 +729,28 @@ func TestCATGood(t *testing.T) {
 	assert.Equal(t, []byte("abcdef"), vm.estack.Peek(0).Bytes())
 }
 
+func TestCATInt0Bytearray(t *testing.T) {
```

Comment:
TestCATInt0Bytearray and TestCATBytearrayInt1: Suggest to use same naming convention and have camel case like:
TestCATInt0ByteArray and TestCATInt1ByteArray
or
TestCATByteArrayInt0 and TestCATByteArrayInt1

Reply:
The order in name mirrors the order of arguments on stack.

Reply:
But `ByteArray` is still supposed to be `ByteArray`, I think.